src/library/scala/collection/GenIterableLike.scala

```diff
@@ -96,20 +96,23 @@ trait GenIterableLike[+A, +Repr] extends Any with GenTraversableLike[A, Repr] {
    *                 result class `That` from the current representation type `Repr`
    *                 and the new element type `(A1, Int)`.
    *  @return        A new collection of type `That` containing pairs consisting of all elements of this
-   *                 $coll paired with their index. Indices start at `0`.
+   *                 $coll paired with their index. Indices start at `indicesStart`.
```

Comment:
I'd suggest renaming `indicesStart` to just `start`.

Reply:
I bluntly copied that name from docs, was said `Indices start at 0`.

Don't mind to change it to `start` or `startFrom` or similar, if community prefer that.